data/github/orgs/opentelekomcloud-infra/repositories/otc-openapi-schema-validator.yml

```diff
@@ -0,0 +1,19 @@
+---
+otc-openapi-schema-validator:
+  default_branch: main
+  description: Configurable validator/linter for OpenAPI schema documents
+  homepage: null
+  archived: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  delete_branch_on_merge: true
+  allow_merge_commit: false
+  allow_squash_merge: true
+  allow_rebase_merge: false
+  teams:
+    - slug: "eco"
+      permission: "push"
+  branch_protections:
+    - branch: "main"
+      template: "zuul"
```

Comment:
do you want to use zuul for ci/cd?

Reply:
yes, better to change, now github-actions